src/nerve/core/nodes/terminal/claude_wezterm_node.py

```
@@ -157,6 +169,15 @@ async def create(
 
             await asyncio.sleep(0.5)
 
+            # If proxy_url is set, export ANTHROPIC_BASE_URL before running claude
+            if proxy_url:
+                export_cmd = f"export ANTHROPIC_BASE_URL={proxy_url}"
+                await inner.backend.write(export_cmd)
+                await asyncio.sleep(0.1)
+                await inner.backend.write("\r")
+                await asyncio.sleep(0.3)  # Wait for export to complete
+                logger.debug(f"Set ANTHROPIC_BASE_URL={proxy_url} for node '{id}'")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)